src/language/typescript/2.0/serializers/responses-object.ts

```diff
@@ -18,9 +18,16 @@ import { fromString, Ref } from '../../../../utils/ref';
 import { ReferenceObjectCodec } from '../../../../schema/3.0/reference-object';
 import { some } from 'fp-ts/lib/Option';
 
-export const serializeOperationResponses = (from: Ref, responses: ResponsesObject): Either<Error, SerializedType> =>
+export const serializeOperationResponses = (
+	from: Ref,
+	responses: ResponsesObject,
+	filter: (code: string) => boolean = () => true,
```

Comment:
no need to pass filter, pass already filtered responses

Reply:
done